jobs/config.json

```diff
@@ -4509,6 +4509,22 @@
       "UNKNOWN"
     ]
   }, 
+  "ci-kubernetes-e2e-kubeadm-gce-1-7": {
+    "args": [
+      "--cluster=", 
+      "--env-file=jobs/platform/gce.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-kubeadm-gce-1-7.env", 
+      "--kubeadm=ci", 
+      "--deployment=kubernetes-anywhere", 
+      "--mode=local", 
+      "--timeout=300m", 
+      "--extract=ci/latest-1.7"
+    ], 
+    "scenario": "kubernetes_e2e", 
+    "sigOwners": [
+      "UNKNOWN"
```

Comment:
made sig-cluster-lifecycle owners of all kubeadm jobs. It's not used by anything yet but yes it's good to set up properly.